sixpack-java/src/main/java/com/seatgeek/sixpack/ParticipatingExperiment.java

```diff
@@ -36,7 +36,20 @@
      * @throws ConversionError if the client fails to convert
      */
     public ConvertedExperiment convert() {
-        return sixpack.convert(this);
+        return sixpack.convert(this, null);
+    }
+
+    /**
+     * Converts this experiment by notifying the Sixpack server that the user has completed the
+     * expected action
+     *
+     * This call makes blocking network requests.
+
+     * @param kpi The kpi of the conversion.
+     * @return
```

Comment:
Please complete the documentation here about the return type and the potential to throw